src/elm/App/Update.elm

```diff
@@ -21,6 +21,10 @@ type Action
   -- NoOp actions
   | NoOp
 
+dragActions : Signal Action
+dragActions =
+    Signal.map ChildTimelineAction Timeline.Update.startTimeActions
```

Comment:
I see.. Yeah, in a certain point I'd like to experiment with arbitrary draggable items. I've also noticed that the current draggable can be moved by clicking anywhere over the bar -- not only over the triangle itself.

Well, experiments are fun :)